tests/Predis/Command/RawCommandTest.php

```diff
@@ -74,10 +74,29 @@ public function testExceptionOnMissingCommandID()
      * argument is missing, PHP emits an E_WARNING.
      *
      * @group disconnected
-     * @expectedException \PHPUnit_Framework_Error_Warning
      */
     public function testPHPWarningOnMissingCommandIDWithStaticCreate()
     {
+        if (version_compare(PHP_VERSION, "7.1", '>')) {
+            $this->markTestSkipped('only for PHP < 7.1');
```

Comment:
Actually I'd prefer phpunit's annotation `@require PHP 7.1` instead of using `version_compare()` + `markTestSkipped()`.